packages/babel-plugin-minify-constant-folding/__tests__/fixtures/member-access/actual.js

```diff
@@ -0,0 +1,13 @@
+({foo: 'oof', bar: 'rab'}).foo;
+({bar: 'rab'}).foo;
+({['foo']: 'oof', bar: 'rab'}).foo;
+({['foo']: 'oof', bar: 'rab'}).baz;
+({bar: 'rab'}).toString;
```

Comment:
It's an edge but this one can be unsafe. Nothing prevents the user from overriding `Object.prototype.toString`.

Reply:
Good catch, I pushed a commit with a fix and more tests